modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/numeric/NumericDDMFormFieldTemplateContextContributorTest.java

```diff
@@ -94,16 +89,18 @@ public void testGetDataTypeChanged() {
 		DDMFormFieldRenderingContext ddmFormFieldRenderingContext =
 			new DDMFormFieldRenderingContext();
 
+		ddmFormFieldRenderingContext.setLocale(_locale);
 		ddmFormFieldRenderingContext.setProperty(
 			"changedProperties",
 			HashMapBuilder.<String, Object>put(
 				"dataType", "double"
 			).build());
 
-		Assert.assertEquals(
-			"double",
-			_numericDDMFormFieldTemplateContextContributor.getPropertyValue(
-				ddmFormField, ddmFormFieldRenderingContext, "dataType"));
+		Map<String, Object> parameters =
```

Comment:
Shouldn't maps variables have the suffix `Maps`? [Reference](https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752519/Java+conventions+in+Liferay#[inlineExtension]Maps-are-plurals,-with-suffix).